css-colors/styles.css

```diff
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+section {
+  background-color: #ade6e6;
+  padding: 1.5rem 0;
+}
+
+h2 {
+  margin-bottom: 3.5rem;
+  font-size: 2rem;
+  text-align: center;
+  font-weight: 300;
+}
+
+form {
+  margin: 0 auto 2rem;
+  text-align: center;
+}
+
+input {
+  border-color: lightgray;
+  width: 400px;
+  padding: 0.5rem;
+  border: 1px solid;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+}
+
+button {
+  padding-left: 1.25rem;
+  padding-right: 1.25rem;
+  margin-left: 0.25rem;
+  border: none;
+  font-weight: 500;
+  color: white;
+  background-color: rgb(166, 255, 0);
+  height: 3rem;
+  border-radius: 0.25rem;
+  font-size
```

Comment:
Whoops! you seem to have an incomplete declaration here. Either remove it or complete it, please :grin: